src/components/ReviewsItemOfList.jsx

```diff
@@ -1,15 +1,24 @@
-const ReviewsItemOfList = ({review}) => {
-    const fullDate = new Date(review.created_at);
-
-return (
-    <li className="reviewsListItem">
-        <h3 className="individualReviewTitle">{review.title}</h3>
-        <p>Author: {review.owner}</p>
-        <p>Posted: {fullDate.getFullYear()}</p> 
-        <p>Votes: {review.votes}</p>
-        <img className="individualReviewImage" src={review.review_img_url}/>
+import { useNavigate } from "react-router";
+
+const ReviewsItemOfList = ({ review }) => {
+  const navigate = useNavigate();
+
+  const clickReview = () => {
+    const path = `/reviews/${review.review_id}`;
+    navigate(path);
+  };
+
```

Comment:
We don't really want to use useNavigate here, because we only use it for side effects.
Say you have a recipe website - after a user submits a recipe, you want to re-direct them to the page for the new recipe. That is when we'd use useNavigate.

If we just click on a link, so the re-direction is NOT a side effect but a deliberate effect, then we want to link to it